### PR TITLE
OCLOMRS-456: Use the traditional OCL url constant defined in HelperFunction.js to replace the traditional url scattered in the codebase.

### PR DESCRIPTION
**Pull request by alexkayabula**
# JIRA TICKET NAME:
[Use the traditional OCL url constant defined in HelperFunction.js to replace the traditional url scattered in the codebase.](https://issues.openmrs.org/browse/OCLOMRS-456)

# Summary:
The URL(https://qa.openconceptlab.org) to the traditional OCL site is scattered in the code base as seen in the attached screenshot. A constant defined in HelperFunction.js should be used instead.

### CODE REVIEW

File: src/tests/Dictionary/DictionaryModal.test.jsx [hunk 1]
```diff
@@ -189,6 +189,52 @@ describe('Test suite for dictionary modal', () => {
     expect(wrapper3.instance().state.errors).toEqual({});
   });
 
+  it('it should handle undefined submit error response', async () => {
```
Comment (collinewait) [inline src/tests/Dictionary/DictionaryModal.test.jsx]: Is this related to the ticket?

Comment (alexkayabula) [reply src/tests/Dictionary/DictionaryModal.test.jsx]: Not exactly, after implementing the changes, the test coverage for that particular test dropped. I had to write that test to increase the coverage.

**Comment by dkayiwa** [comment]
@alexisduque why is this pull request open for now close to a week?

**Comment by alexkayabula** [comment]
> @alexkayabula  why is this pull request open for now close to a week?

@dkayiwa, I sent you a request to merge this pull request over the weekend.

**Comment by dkayiwa** [comment]
You need to keep pinging me if you do not get a response from me for a day.

**Comment by alexkayabula** [comment]
> You need to keep pinging me if you do not get a response from me for a day.

My apologies @dkayiwa noted.

**Comment by dkayiwa** [comment]
@alexkayabula and what have you been working on since then?

**Comment by alexkayabula** [comment]
> @alexkayabula and what have you been working on since then?

I am currently working on [OCLOMRS-460](https://issues.openmrs.org/browse/OCLOMRS-460).

File: src/tests/Dictionary/DictionaryModal.test.jsx [hunk 2]
```diff
@@ -189,6 +189,52 @@ describe('Test suite for dictionary modal', () => {
...
+        description: 'OpenMRSDictionary',
+        default_locale: 'en',
+        supported_locales: 'us',
+        repository_type: 'OpenMRSDictionary',
+      },
+      errors: {},
+    });
+
+    await wrapper3.instance().onSubmit(preventDefault);
+    expect(wrapper3.instance().state.errors).toEqual({});
```
Comment (dkayiwa) [inline src/tests/Dictionary/DictionaryModal.test.jsx]: What is the above testing?

Comment (alexkayabula) [reply src/tests/Dictionary/DictionaryModal.test.jsx]: We are testing for a scenario where the error returned in the response when adding a dictionary is not defined.
